### ISSUE TITLE
[BUG] Can't create passkeys

### ISSUE THREAD

**Issue by RubberDuckShobe** [AChep/keyguard-app#719]
**Describe the bug**
I am trying to create a Passkey with Keyguard set as the primary password manager in the Android settings, but Firefox Nightly asks if I want to store the passkey in my Google account and Stable Firefox asks me if I want to use another device.

**To Reproduce**
1. Set Keyguard as the preferred password manager in Android's settings
2. Open a site that supports passkeys (GitHub in my case)
3. Try to create a passkey

**Expected behavior**
The prompt should ask about creating a passkey with Keyguard

**Screenshots**
![Screenshot_20241127-151821_Credential Manager~2](https://github.com/user-attachments/assets/4e52e219-5868-4939-9f4e-5da725db07b1)

**Smartphone:**
 - Device: OnePlus 7T
 - OS: Android 14
 - Browser Firefox and Firefox Nightly
 - App version 1.6.4

**Additional context**
I don't have premium yet. (Is passkey creation a premium feature? If so, I don't think the readme/app makes it clear enough)

I use a custom ROM (crDroid 10.10)

Bitwarden Beta works (except on Firefox Nightly, because it's not on the list of "privileged browsers")

**Comment by AChep** [comment]
_Hmm, I should really revisit showcasing locked features..._ Yeah, any vault modification is a premium feature, including adding/removing passkeys. You can test if the version from GitHub works.

**Comment by RubberDuckShobe** [comment]
Yep, the GitHub version works. Though signing in on GitHub with the passkey fails... But I guess that's for a different issue
